transaction-scenarios/src/scenarios/radiswap.rs

```diff
@@ -172,28 +227,32 @@ impl ScenarioCreator for RadiswapScenarioCreator {
                                 code.to_vec(),
                                 schema,
                                 metadata_init! {
-                                    "name" => "Radiswap Package".to_owned(), locked;
-                                    "description" => "A package of the logic of a Uniswap v2 style DEX.".to_owned(), locked;
-                                    "tags" => vec!["dex".to_owned(), "pool".to_owned(), "radiswap".to_owned()], locked;
+                                    "name" => "Radiswap Package", locked;
+                                    "description" => "[EXAMPLE] A package of the logic of a Uniswap v2 style DEX.".to_owned(), locked;
```

Comment:
Minor: this `ToOwned` is no longer needed.

Reply:
Yes - there's loads of fixing possible now across the code base thanks to my fixing up of ToMetadataEntry :D